README.md

```diff
@@ -99,6 +102,57 @@ function App() {
 }
 ```
 
+### Elements
+
+Unform exposes two elements by default, Input and Select.
+
+#### Input element
+
+Input elements can receive a `multiline` prop that will make it act as a textarea.
```

Comment:
`...prop that will make it act as a textarea` -> `...prop that will render a textarea instead`

Maybe this way is better, because it is completely clear that when `multiline` is passed as true, a `<textarea>` will render instead of an `<input>`.

But this is me being real nitpicky, which means this is optional. Its up to you to change it or not. 😄